src/main.rs

```diff
@@ -424,15 +424,17 @@ async fn main() -> Result<()> {
         warn!("INSECURE: Only use Keylime in this mode for testing or debugging purposes.");
     }
 
-    // Verify if the python shim is installed in the expected location
-    let python_shim =
-        PathBuf::from(&config.revocation_actions_dir).join("shim.py");
-    if !python_shim.exists() {
-        error!("Could not find python shim at {}", python_shim.display());
-        return Err(Error::Configuration(format!(
-            "Could not find python shim at {}",
-            python_shim.display()
-        )));
+    if cfg!(feature = "legacy-python-actions") {
```

Comment:
What about using [`cfg_if!`](https://docs.rs/cfg-if/1.0.0/cfg_if/) to compile out unnecessary code?

Reply:
OK, I'll modify the code to use `cfg_if!`